### PR TITLE
Restore staging to before the unintentional revert.

### PR DESCRIPTION
**Pull request by kacf**
1. Merges master into staging right before the unintentional commit.
2. Merges the unintentional commit using merge strategy "ours" (to keep the staging content)
3. Merges the rest of the changes since the unintentional commit using normal merge.

### CODE REVIEW

**Review by tranchitella — APPROVED** [review]
Diff looks good!

File: tests/tests/test_update_modules.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def make_artifact(artifact_file, artifact_id):
             deploy.check_expected_statistics(deployment_id, "failure", 1)
             deploy.check_expected_status("finished", deployment_id)
 
-            output = mender_device.run("mender -show-artifact").strip()
+            output = mender_device.run("mender -no-syslog -show-artifact").strip()
```
Comment (lluiscampos) [inline tests/tests/test_update_modules.py:68]: Changes on this file I think are the result of a bad conflict resolution at some point. (It was incorrect before my mess, so I'll fix independently later on)

**Review by lluiscampos — APPROVED** [review]
Looks good. Thank you, Kristian.

**Comment by kacf** [comment]
Regarding testing, do I use the regular "start pipeline" approach, or does this need special treatment?

**Comment by tranchitella** [comment]
You can ask @mender-test-bot to start pipeline, right :)

**Comment by lluiscampos** [comment]
> Regarding testing, do I use the regular "start pipeline" approach, or does this need special treatment?

Regular pipeline, as Fabio has triggered. The only special thing is that we expect some tests to fail (related to unreleased features from the client).

**Comment by lluiscampos** [comment]
Looks good. Merging

**Pull request merged**